Makefile

```diff
@@ -0,0 +1,16 @@
+default: fmt
+
+dep:
+	npm install
+
+fmt:
+	npx prettier --write .
+
+clean:
+	@echo "Nothing to clean"
+
+check:
+	@echo "Nothing to check"
+
+build:
+	npm run build
```

Comment:
```suggestion
	npm run build
	
```

Reply:
?

Reply:
just keeping empty line in the end as we usually do

Reply:
Ok